tests/test_lib.py

```diff
@@ -7,7 +8,9 @@ def test_1() -> None:
 
 
 def test_run_uv_nox() -> None:
-    folder = Path(__file__).parent
-    noxfile = folder / "subnoxfile.py"
-    a = subprocess.run(["python3", "-m", "nox", "-f", f"{noxfile}"])
+    cur_folder = Path.cwd()
+    testing_folder = Path(__file__).parent / "subproject"
+    os.chdir(testing_folder)
+    a = subprocess.run(["python3", "-m", "nox"])
```

Comment:
This isn't especially portable to Windows, which uses `py` launcher instead.

Could this be `uv run python -m nox` instead?